medicai/models/convnext/convnext_v2.py

```
@@ -111,6 +117,14 @@ def __init__(
         if include_top:
             x = GlobalAvgPool(x)
             x = GlobalNorm(x)
+
+            VALID_ACTIVATION_LIST = keras_constants.get_valid_activations()
+            if classifier_activation not in VALID_ACTIVATION_LIST:
+                raise ValueError(
+                    f"Invalid value for `classifier_activation`: {classifier_activation!r}. "
+                    f"Supported values are: {VALID_ACTIVATION_LIST}"
+                )
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)